sljit_src/sljitExecAllocator.c

```diff
@@ -189,7 +200,10 @@ static SLJIT_INLINE void free_chunk(void *chunk, sljit_uw size)
 	munmap(chunk, size);
 }
 
-#endif
+#endif /* windows */
+#ifndef SLJIT_UPDATE_WX_FLAGS
+#define SLJIT_UPDATE_WX_FLAGS(from, to, enable_exec)
+#endif /* !SLJIT_UPDATE_WX_FLAGS */
```

Comment:
alternatively, this could be defined in the windows code to avoid the awkward duplicate ifndef